src/main/java/com/cloudbees/jenkins/plugins/awscredentials/AWSCredentialsImpl.java

```diff
@@ -202,6 +208,37 @@ private static AssumeRoleRequest createAssumeRoleRequest(String iamRoleArn) {
                 .withRoleSessionName("Jenkins");
     }
 
+    /**
+     * Provides the {@link AWSSecurityTokenService} for a given {@link AWSCredentials}
+     * @param awsCredentials
+     *
+     * @return {@link AWSSecurityTokenService}
+     */
+    private static AWSSecurityTokenService getAWSSecurityTokenService(AWSCredentials awsCredentials) {
+        ClientConfiguration clientConfiguration = getClientConfiguration();
+        return AWSSecurityTokenServiceClientBuilder.standard()
+                .withCredentials(new AWSStaticCredentialsProvider(awsCredentials))
+                .withClientConfiguration(clientConfiguration)
+                .build();
+    }
+
+    /**
+     * Provides the {@link ClientConfiguration}
+     *
+     * @return {@link ClientConfiguration}
+     */
+    private static ClientConfiguration getClientConfiguration() {
+        ProxyConfiguration proxy = Jenkins.getInstanceOrNull().proxy;
```

Comment:
The project doesn't build because of that.

_possible null pointer dereference in com.cloudbees.jenkins.plugins.awscredentials.AWSCredentialsImpl.getClientConfiguration() due to return value of called method [com.cloudbees.jenkins.plugins.awscredentials.AWSCredentialsImpl, com.cloudbees.jenkins.plugins.awscredentials.AWSCredentialsImpl] Dereferenced at AWSCredentialsImpl.java:[line 231]Known null at AWSCredentialsImpl.java:[line 231] NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE_

 As Jenkins.get() is not available until 2.98, null check might be performed.